Source/JavaScriptCore/inspector/InjectedScriptBase.cpp

```diff
@@ -41,14 +42,77 @@
 
 namespace Inspector {
 
+static RefPtr<JSON::Value> jsToInspectorValue(JSC::JSGlobalObject* globalObject, JSC::JSValue value, int maxDepth)
+{
+    if (!value) {
+        ASSERT_NOT_REACHED();
+        return nullptr;
+    }
+
+    if (!maxDepth)
+        return nullptr;
+
+    maxDepth--;
+
+    if (value.isUndefinedOrNull())
+        return JSON::Value::null();
+    if (value.isBoolean())
+        return JSON::Value::create(value.asBoolean());
+    if (value.isNumber() && value.isDouble())
+        return JSON::Value::create(value.asNumber());
+    if (value.isNumber() && value.isAnyInt())
+        return JSON::Value::create(static_cast<int>(value.asAnyInt()));
```

Comment:
And this should be `value.asInt32()` without a `static_cast`.

Reply:
Changed.